src/index.js

```diff
@@ -0,0 +1,111 @@
+import './style.css';
+
+const showList = document.getElementById('todoList');
+
+const LoadToDoList = () => {
+  const containerToDoList = document.createElement('div');
+  containerToDoList.innerHTML = `
+<p id="header">Today's To Do <i class="fas fa-sync"></i></p>
+      <input class="data-entry" type="text" placeholder="Add to your list..." required></input>
+`;
+  showList.appendChild(containerToDoList);
+};
+
+LoadToDoList();
+
+const AddTaskToList = (task, index, done) => {
+  const containerTasks = document.createElement('div');
+  containerTasks.innerHTML += `
+        <input type="checkbox" id='${index}' class="checkbox">
+        <input type='text' value='${task}' class='task ${done ? 'done-task' : ''}'>
+        <i class="fas fa-ellipsis-v"></i>
+        <i class="fas fa-trash-alt" id='${index + 1}'></i>
+      `;
+  showList.appendChild(containerTasks);
+};
+
```

Comment:
- [ ] Great job so far 👏🏻 I can see that you have to implement the CRUD (create, update, delete) methods 😜 💪🏻  however for this Milestone you were only asked to create an array of some simple *to do tasks* (array of objects) with each object containing three keys - description \[`string`\], completed \[`bool`\], index: \[`number`\]. 
eg:

   ```
   const tasks = [
     {
       description: 'An example description in string',
       completed: false,
       index: 10,
     },
   ];
   ```

- [ ] After that you should implement a function that iterates over the tasks array and populates the HTML list item element for each task.

```

const populateTasks = (task, parentElement) => {
  const { description, completed, index: id } = task;
  const innerHTML = `
         <p>${description}</p>
         <p>${completed ? 'completed' : 'not completed'}</p>`;
  const newTodoListElement = document.createElement('li');
  Object.assign(newTodoListElement, { innerHTML, id });
  parentElement.appendChild(newTodoListElement);
};
```

- Kindly address this issue 👨🏻‍🎨

Reply:
@BenMukebo I have the three keys inside the object I just don't use the exact words because I had problems with linters using long lines, it's frustrating to have this type of observations because the purpose is being fulfilled, it's frustrating to see that instead of motivating me to move forward I had to delete my code to meet the requirements of a basic structure. I don't feel that this helps me improve my project.

Reply:
Hello @Alondra87, Sorry for the late reply, I see you have already implemented the requested change congratulations 🎉 🥳
The reason I shouldn't endorse this PR isn't that you haven't done much, just because the main reason for doing this step was to get to `1. create an array of some simple tasks, 2. Implement a function that iterates over the array of tasks and populates the HTML list item element for each task.
but you have gone ahead and implemented the requirements for the next step, which was not requested